man/fi_mr.3.md

```diff
@@ -932,6 +932,14 @@ The follow flag may be specified to any memory registration call.
   fi_mr_attr structure.  This flag is only usable for domains opened with
   FI_HMEM capability support.
 
+*FI_MR_SINGLE_USE*
+: This flag indicates that the memory region is only used for a single
+  operation.  After the operation is complete, the key associated with the
+  memory region is automatically invalidated and can no longer be used for
+  remote access.  This flag is intended to be used when registering a
+  sub-MR to get a single use key of the base MR.  However, it can be used
+  with any memory region.
```

Comment:
Drop text referring to sub-mr.  It applies to any MR, no need to call out an intent.